### DISCUSSION TITLE
Line charts does not display correct information when filtering out issues

### DISCUSSION THREAD

**Discussion by harvinders** [community/community#46911] [Projects and Issues]
### Select Topic Area

Bug

### Body

Take this example below, I have a project and I am ploting all the issues in the project

![image](https://user-images.githubusercontent.com/8557046/217967933-6a8eb519-ff38-413e-bb00-7c14af9841fa.png)

If now I filter all the issues with `done` status. It removes the data points for those issues when they were not in the `done` status too leading to something like

![image](https://user-images.githubusercontent.com/8557046/217968163-5e8aeffa-d22f-447d-9699-8a759d39471b.png)

On 26th, the first chart says that there were ~50 issues in not done state, but the second chart displays only ~35 issues. All those issues which today are in `done` status are removed.

**Comment by chiedo** [comment]
Thanks for reporting this! We'll keep our eye on it and see if we can reproduce the issue.
